README.md

```diff
@@ -55,23 +55,47 @@ To get started:
 1. Add "Post Build Action" -> "Send to Fluentd".
 1. Specify the desired file (and extension in JSON format if needed).
 
-Examples
+Send to Fluentd Post-build Action
```

Comment:
I'm not sure what this is supposed to mean. Maybe "Sending data to Fluentd as a post-build action"?